### ISSUE TITLE
K100 RGB Optical wheel action binding

### ISSUE THREAD

**Issue by Fail-gh** [ckb-next/ckb-next#1037]
### Enhancement Details

Add the binding option to the wheel, now is useless

**Comment by kimjongbing** [comment]
I want to add onto this that the wheel right now (not the button inside the wheel) if slightly touched or spun will cause erratic behavior like the triggering of many keys on the keyboard and page direction (scrolling up/down) and navigating to other pages (back/forward).

**Comment by Azyrion** [comment]
the wheel on the keyboard is definitely useless and also causes a lot of trouble. No idea why but it posted some nonsense in a discord group that nearly kicked me out! So the software is doin something with it.....can you at least unbind it?

**Comment by mmtmn** [comment]
debugging the wheel was a pain, but this was the key insight to figure out what happens when you use the wheel:

KEY_FASTFORWARD (208) → clockwise
KEY_REWIND (168) → counter-clockwise

working on a PR for this

**Comment by mmtmn** [comment]
I've opened a PR on this over at #1170

**Comment by Azyrion** [comment]
any luck @mmtmn ?

**Comment by mmtmn** [comment]
@Azyrion yeah, I fixed it, its in this pull request: https://github.com/ckb-next/ckb-next/pull/1172

no idea why it hasn't been approved or reviewed by @tatokis or @Ravenslofty

**Comment by mmtmn** [comment]
yeah pretty sus open source repository I guess, i've seen other suggest valid PRs and then get pretty badly and unfairly treated, I'm not trusting this anymore, so i'll do what other did and just close it and never use and recommend against using it

**Comment by KABoissonneault** [comment]
It's a bit unfortunate what happened to the PR. This issue is actually quite bad, accidentally hitting that wheel will send random inputs all over. I don't really need support to bind it to anything relevant, I'd like if I could just disable it.

**Comment by Ravenslofty** [comment]
a) you can't disable it, the hardware will send us events whether we want them or not. 
b) did you try building latest git from source?  2103316 should go some way towards resolving it.

**Comment by KABoissonneault** [comment]
Actually, yes. I've tried the -git AUR package just now and was coming back to edit my post. The wheel now (poorly) controls volume, which is good enough for me.

**Comment by Ravenslofty** [comment]
Glad to hear it.

**Comment by MadMartian** [comment]
Mapping it to volume control is better than nothing, currently it crashes [keyd](https://github.com/rvaiya/keyd) with a panic, which I'm using to manage input from a different macropad.  I suppose installing the latest is a better option that dumping a wad of hot plastic glue on the knob so I never accidentally bump it.
